homeassistant/components/rflink/sensor.py

```diff
@@ -96,14 +103,36 @@ class RflinkSensor(RflinkDevice):
     """Representation of a Rflink sensor."""
 
     def __init__(self, device_id, sensor_type, unit_of_measurement,
-                 initial_event=None, **kwargs):
+                 expire_after=0, force_update=None, initial_event=None,
```

Comment:
Should not it be `force_update=False`?
I know it's copied from the `binary_sensor`, but I think it's better to use `False` than `None` (also in `binary_sensor`)

Reply:
After reading entity.py it seems like bool is its type so you're right.
I'll make a separate PR for binary_sensor if it's still required.

Reply:
> After n seconds the sensor's value changes to None so its state becomes unknown.
> When a new reading arrives, the value changes to that value and consequently the sensor's state changes as well.

thanks for the clarification